Gemfile.lock

```diff
@@ -221,17 +221,15 @@ GEM
       rb-fsevent (~> 0.10, >= 0.10.3)
       rb-inotify (~> 0.9, >= 0.9.10)
     mercenary (0.3.6)
+    mini_portile2 (2.8.4)
     minima (2.5.1)
       jekyll (>= 3.5, < 5.0)
       jekyll-feed (~> 0.9)
       jekyll-seo-tag (~> 2.1)
     minitest (5.18.0)
     multipart-post (2.1.1)
-    nokogiri (1.13.8-arm64-darwin)
-      racc (~> 1.4)
-    nokogiri (1.13.8-x64-mingw32)
```

Comment:
기존에는 OS 별로 nokogiri 라이브러리의 설치 버전이 명시되어 있었으나, 의존성을 업데이트 하는 과정중에 nokogiri 라이브러리가 모두 1.13.8버전으로 통합되었습니다. 

각기 다른 OS(Window - mingw, Linux - x86 architecture)에서 빌드를 수행했을 때, 이 부분으로 인한 빌드 충돌이 없는지 확인을 해봐야 할 것 같습니다.

Reply:
다른 OS 에서도 정말 문제가 없는지 확인이 필요하겠군요. 일이 약간 커졌네요 😄 
가능하시면 직접 관련 OS 환경에서 확인해보면 좋겠지만, 어렵다면 확인이 될때까지 보류해야할 것 같아요

Reply:
@bnbong: Would you re-consider this, because this comment part was already addressed through https://github.com/Azure/azure-sdk-korean/pull/116 ?

Reply:
> @bnbong: Would you re-consider this, because this comment part was already addressed through #116 ?

@ianychoi I have checked the contents. Reflecting #116, I have rewritten Gemfile.lock because it seems better to restore the parts related to nokigiri library dependency to the original content.